docs/howto/multiple-storage-backends.md

```diff
@@ -0,0 +1,260 @@
+
+```
+  </div>
+
+  <div markdown="3" id="hybrid">
+
+This hybrid setup allows lakeFS to manage data across both cloud and on-prem storages.
+```yaml
+blockstores:
+    signing:
+      secret_key: "some-secret"
+    stores:
+        - id: "s3-archive"
+          description: "AWS S3 storage for long-term archival"
+          type: "s3"
+          s3:
+            region: "us-west-2"
+        - id: "minio-fast-access"
+          description: "On-prem MinIO for high-performance workloads"
+          type: "s3"
+          s3:
+            force_path_style: true
+            endpoint: 'http://minio.local'
+            discover_bucket_region: false
+            credentials:
+              access_key_id: "minio_access_key"
+              secret_access_key: "minio_secret_key"
+
+```
+  </div>
+</div>
+
+### Key Considerations
+
+* Unique Blockstore IDs: Each storage backend must have a unique id.
+* Persistence of Blockstore IDs: Once defined, an id must not change.
+* S3 Authentication Handling:
+    * All standard S3 authentication methods are supported.
+    * If static credentials are provided, lakeFS will use them. Otherwise, it will fall back to the AWS credentials chain. 
+      This means that for setups with multiple storages of type `s3`, static credentials are required for all but one.
+
+### Upgrading from Single to Multi-Store
+
+When upgrading from a single storage backend to a multi-store setup, follow these guidelines:
+* Use the new `blockstores` structure, **replacing** the existing `blockstore` configuration. Note that `blockstore` and `blockstores` 
+  configurations are mutually exclusive - lakeFS does not support both simultaneously. 
+* Define all previously available [single-blockstore settings](../reference/configuration.md#blockstore) under their respective storage backends.
+* The `signing.secret_key` remains a required global setting.
+* Set `backward_compatible: true` for the existing storage backend to ensure:
+  * Existing repositories continue using the original storage backend.
+  * Newly created repositories default to this backend unless explicitly assigned a different one, to ensure a non-breaking upgrade process. 
+
+### Adding or Removing a Storage Backend
+
+To add a storage backend, update the server configuration with the new storage entry and restart the server.
+
+To remove a storage backend:
+* Delete all repositories associated with the storage backend.
+* Remove the storage entry from the configuration.
+* Restart the server.
+
+{: .warning}
+> Repositories linked to a removed storage backend will result in unexpected behavior. Ensure all necessary cleanup is done before removal.
+
+### Listing Connected Storage Backends
+
+The [Get Config](https://docs.lakefs.io/reference/api.html#/config/getConfig) API endpoint now returns a list of storage
+configurations. In multi-store setups, this is the recommended method to list connected storage backends and view their details.
```

Comment:
Not sure if you want to be concrete here, but the API returns both a string and a list and the user should use the relevant